openshift2nulecule/utils.py

```diff
@@ -201,3 +201,27 @@ def run_cmd(cmd, checkexitcode=True, stdin=None):
             raise Exception("cmd: %s failed: \n%s" % (str(cmd), stderr))
 
     return ec, stdout, stderr
+
+
+def get_new_name(filepath):
+    """
+    If filepath exists get new one that doesn't.
+    Changes filename of file in path.
+    Example:
+      if /foo/bar.json exists retruns /foo/bar_1.json
+
+    Args:
+      filepath - existing filepath
+
+    Returns:
+      str - new filepath that is not existing.
+
+    """
+    base, ext = os.path.splitext(filepath)
+    new_filepath = filepath
+    i = 0
+    while os.path.exists(new_filepath):
+        i += 1
+        #  create new path with index /foo/bar_1.json
+        new_filepath = "{}_{}{}".format(base, i, ext)
```

Comment:
Here instead of `i` we can have `next(i)` and line#224 which is `i += 1` can be removed.